task/prepare.go

```diff
@@ -0,0 +1,80 @@
+package task
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/golang/glog"
+	livepeerAPI "github.com/livepeer/go-api-client"
+	"github.com/livepeer/go-livepeer/drivers"
+	"github.com/livepeer/stream-tester/model"
+	"github.com/livepeer/stream-tester/segmenter"
+)
+
+func RecordStream(ctx context.Context, lapi *livepeerAPI.Client, playbackID string, osSess drivers.OSSession) (string, error) {
+	glog.Infof("made it into record function")
+	fullPath := videoFileName(playbackID)
+	glog.Infof("full path %s", fullPath)
+	fir, err := osSess.ReadData(ctx, fullPath)
```

Comment:
what does `fir` stand for here?

Reply:
https://github.com/livepeer/task-runner/blob/main/task/transcode.go#L77

`FileInfoReader`

Go's insistence on extra short variable names is bizarre to me 😂

Reply:
I think that's more coding style than go's general pattern. I'm down for using more descriptive names everywhere 👀

Reply:
Eli mentioned it is idiomatic in Go to use shorter/short as possible variable names, this article makes the distinction to do so proportionately to scope which is insightful 
https://murfitt.net/blog/short-variable-names-in-go

Reply:
Right, so I think I also do use shorter names when it is a method receiver or very common arguments like `*http.Request`, `http.ResponseWriter`, reader, writer etc. I don't think it is the case here since I don't have `FileInfoReaders` all around I guess. Although it **is** a reader and is also only used close to where it is declared anyway, so not sure... I guess it's kinda subjective in the end. This quote as well:

> More unusual things and global variables need more descriptive names.

IMO if you need an IDE to know what is a thing or keep having to go back to its definition (enters that "scope" insight), you might as well use a more descriptive name. In this case, IMO it's not obvious that `ReadData` returns something called `FileInfoReader`, so a descriptive name would help. If it were called `OpenFileReader`, `ReadFile` or sth like that, then it could make more sense.

But this is a small nit anyway. I wouldn't pro-actively ask to rename this, but since we're already talking/discussing about it that is my perspective on short var names.